clang/docs/HLSL/ExpectedDifferences.rst

```diff
@@ -108,3 +108,16 @@ behavior between Clang and DXC. Some examples include:
   diagnostic notifying the user of the conversion rather than silently altering
   precision relative to the other overloads (as FXC does) or generating code
   that will fail validation (as DXC does).
+
+Correctness improvements (bug fixes)
+====================================
+
+Entry point functions & ``static`` keyword
+------------------------------------------
+Marking a shader entry point function ``static`` will result in an error.
```

Comment:
Is there a test for this already?

Reply:
Yes: https://github.com/llvm/llvm-project/blob/e80c59556d2d71cc2d0dcb2bd712c36cc4043025/clang/test/SemaHLSL/shader_type_attr.hlsl#L20-L22

Reply:
Cool.  Is it significant that these are warnings, while the rst file says errors?